app/models/like.rb

```diff
@@ -1,2 +1,13 @@
 class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User  '
```

Comment:
- [ ] You did a great job working on this milestone! 💯  However, please be extra careful when using strings, as empty spaces do count as characters, so you would have problems with this one! 🤓 🔧 

```suggestion
  belongs_to :author, class_name: 'User'
```

Reply:
thank you, i can't believe i missed that